### PR TITLE
Rewrote README to be clearer

### PR DESCRIPTION
**Pull request by taesup**
Dropped all reference to how this is a game because it consistently confused students. Instead it's been turned into a tracking system that just keeps track of how many times a specific buzzword was used in a conversation. The "system" can be loaded with up to 5 buzzwords. The "PUT" method is used as a way to mark that a given buzzword was heard and the points should be updated accordingly. The "PUT" response should be the buzzword itself along with the new total score. All other aspects of the exercise was not modified.

### CODE REVIEW

**Review by jaywon — COMMENTED** [review]
LGTM ✅ 

Would be good if another instructor who has gone through this exercise can give ultimate approval. 

Thanks for improving it tho 💯

File: README.md [hunk 1]
```diff
@@ -30,36 +29,36 @@ You will store these objects in memory for the time being. Meaning that if the *
...
-| `POST /buzzword` | `{ "buzzWord": String, "points": Number }` | `{ "success": true }` | Creates a new buzzword object. Returns `true` if successful else `false`|
-| `PUT /buzzword` | `{ "buzzWord": String, "heard": Bool }` |  `{ "success": true, newScore: Number }` | Updates a buzzword. Returns `true` and the new score if successful otherwise returns just `false` |
-| `DELETE /buzzword` | `{ "buzzWord": String }` | `{ "success": true }` | Delete a buzzword. Returns `true` if successful else `false` |
-| `POST /reset` | `{ "reset": true }` | `{ "success": true }` | Resets the server. All buzzwords are removed and scores reset to `0` |
+| `POST /buzzwords` | `{ "buzzWord": String, "points": Number }` | `{ "success": true }` | Creates a new buzzword object. Returns `true` if successful else `false`|
+| `PUT /buzzwords` | `{ "buzzWord": String, "heard": Bool }` |  `{ "success": true, newScore: Number }` | Updates a buzzword's heard value. This also marks that a buzzword has been heard and should update the total score. Returns `true` and the new total score if successful otherwise returns just `false` |
```
Comment (jesse-copeland) [inline README.md]: `PUT` to `/buzzword` should modify the existing buzzword model, not score the buzzword. A new endpoint should be created for that.
`newScore:` should be changed to `totalScore`

File: README.md [hunk 2]
```diff
@@ -30,36 +29,36 @@ You will store these objects in memory for the time being. Meaning that if the *
...
   Example:
-  if `{ "buzzWord": "Social-Mobile", "heard": true }` is sent to this route then the server will find the buzz word "Social-Mobile" and change it's `heard` property to `true`.
+  if `{ "buzzWord": "Social-Mobile", "heard": true }` is sent to this route then the server will find the buzz word "Social-Mobile" and change it's `heard` property to `true`. It should then update the total score kept on the server and return that value as part of the response object.
 
-`DELETE /buzzword`: Deletes a buzz word from the collection.
+`DELETE /buzzwords`: Deletes a buzz word from the collection.
 
 ## Score?
-For now, your server only handles one user's game and one user's score total. It's up to you on how you keep track of the user's score.
+For now, it's up to you on how you keep track of the user's score.
```
Comment (jesse-copeland) [inline README.md]: A new route should be added the test if a buzzword has been heard.
`PUT` to `/heard` by passing the `{"buzzWord": "Social-Mobile"}, if "Social-Mobile" is a buzzWord that was created using with `POST` to `/buzzwords` then the totalScore would be incremented by the point value of the buzzword, and `{"success": true, "totalScore": 'some inc value'}` will be returned.

File: README.md [hunk 3]
```diff
@@ -30,36 +29,36 @@ You will store these objects in memory for the time being. Meaning that if the *
...
 
-`POST /buzzword`: Creates a new buzz word object. The body **should** have these keys:
+`POST /buzzwords`: Creates a new buzz word object. The body **should** have these keys:
   - `buzzWord` which contains the buzz word as a `String` and
   - `points` property is how many points that word is worth when scored, this value is of data-type `Number`.
   Example:
   if `{ "buzzword": "Agile is amazing", "score": 1000 }` is sent to this route then the server will create a new [buzz word object](https://github.com/expressjs/body-parser#bodyparserurlencodedoptions) and add it to a collection.
 
-`PUT /buzzword`:  Updates a buzz word's `heard` property. The body **should** contain these keys:
+`PUT /buzzwords`:  Updates a buzz word's `heard` property. This also marks that the buzzword has been "heard" in the conversation. Therefore, the total score should be incremented with the score value attached to the heard buzzword. The body **should** contain these keys:
```
Comment (jesse-copeland) [inline README.md]: `heard` should not be a property of a buzzWord

**Review by jesse-copeland — CHANGES_REQUESTED** [review]
We need to clarify how a buzzword is scored

**Pull request merged**